### PR TITLE
fix typos: Ok and Check-In (closes #3120) (COMMUNITY)

### LINKED ISSUE #3120: typos: Ok, Okay and Check-In in german translation
## Describe the bug

I found a few more typos: Ok and Okay and Check-In in german translations.

## Expected behaviour

For conistency reasons it should be "OK" and "Check-in", like everywhere else.

## Possible Fix

Just change the texts.

---
Internal Tracking ID: [EXPOSUREAPP-7148](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7148)

### PR DESCRIPTION
**Pull request by DerVogel2020**
fix typos for consistency, see #3120 

---
Internal Tracking ID: [EXPOSUREAPP-7148](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7148)

### CODE REVIEW

**Comment by d4rken** [comment]
@DerVogel2020 Thanks. 
Typos are always :frowning_face:.
Fortunately none of these are critical enough to require a hotfix for 2.2.
Could you retarget your changes against `release/2.3.x`?

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -467,7 +467,7 @@
     <!-- YMSI: onboarding(tracing) - dialog about manual checking -->
     <string name="onboarding_manual_required_dialog_body">"Beachten Sie, dass Sie ohne Aktivierung der priorisierten Hintergrundaktivität die App mindestens alle 24 Stunden einmal aktiv aufrufen müssen, um Ihren Risikostatus aktualisieren zu können.\n\nSie können die priorisierte Hintergrundaktivität jederzeit in Ihren Einstellungen aktivieren."</string>
     <!-- XBUT: onboarding(tracing) - dialog about manual checking button -->
-    <string name="onboarding_manual_required_dialog_button">"Ok"</string>
+    <string name="onboarding_manual_required_dialog_button">"OK"</string>
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:470]: Note that most buttons automatically capitalize the label, and should  usually not be capitalized within the resource files as it reduces their re-usability.

**Comment by dsarkar** [comment]
@DerVogel2020 Thanks. See  @d4rken 's comment on capitalising the label.

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (DerVogel2020) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:470]: Ok, I will change it.
Just thought of consistency of the other "OK"-texts. I will check them too.

**Review by ChristinaTsigganos — DISMISSED** [review]
UA approved, thank you very much for your feedback, @DerVogel2020!

**Review by DerVogel2020 — COMMENTED** [review]
Removed the capitalisation of the "OK"-Button in the duration picker dialog.

**Pull request merged**